cmd/csaf_downloader/downloader.go

```diff
@@ -92,9 +92,17 @@ func (d *downloader) addStats(o *stats) {
 	d.stats.add(o)
 }
 
+// checkRedirect specifies the policy for handling redirects.
+func checkRedirect(req *http.Request, via []*http.Request) error {
```

Comment:
Use `_` for unused parameters.
```
func checkRedirect(req *http.Request, _ []*http.Request) error
```

Reply:
I would agree to the new name for the function if it would just be there to create the log but the purpose of the callback of the client is to check if it should follow a redirect.
We could:

- name the function `logRedirect` nonetheless
- create a new function `logRedirect` which is called from `checkRedirect` (which would be a very small function and therefore I can understand if this is not a good solution)
- stay with the current name (but maybe add a comment for the logging part)

Reply:
As soon as there is some actual checking we can name it check... For now there is only the logging.
so I would recommend to name it log... .